docs/src/css/custom.css

```diff
@@ -0,0 +1,519 @@
+/*
+  Slighlty modified version of https://github.com/ignite/cli/blob/develop/docs/src/css/custom.css
```

Comment:
should this comment note the modifications? If there are upstream changes we want to integrate, it'd help a lot to know what was modified and why